yb-voyager/cmd/analyzeSchema.go

```diff
@@ -1015,12 +1014,46 @@ func analyzeSchemaInternal(sourceDBConf *srcdb.Source) utils.SchemaReport {
 			checkExtensions(sqlInfoArr, filePath)
 		}
 		checker(sqlInfoArr, filePath)
+
+		if objType == "CONVERSION" {
+			checkConversions(sqlInfoArr, filePath)
+		}
 	}
 
 	schemaAnalysisReport.SchemaSummary = reportSchemaSummary(sourceDBConf)
 	return schemaAnalysisReport
 }
 
+func checkConversions(sqlInfoArr []sqlInfo, filePath string) {
+	for _, sqlStmtInfo := range sqlInfoArr {
+		parseTree, err := pg_query.Parse(sqlStmtInfo.stmt)
+		if err != nil {
+			utils.ErrExit("failed to parse the stmt %v: %v", sqlStmtInfo.stmt, err)
+		}
+
+		createConvNode, ok := parseTree.Stmts[0].Stmt.Node.(*pg_query.Node_CreateConversionStmt)
+		if ok {
+			createConvStmt := createConvNode.CreateConversionStmt
+			//Conversion name here is a list of items which are '.' separated 
+			//so 0th and 1st indexes are Schema and conv name respectively
+			nameList := createConvStmt.GetConversionName()
+			convName := nameList[0].GetString_().Sval
+			if len(nameList) > 1 {
+				convName = fmt.Sprintf("%s.%s", convName, nameList[1].GetString_().Sval)
+			} 
+			reportCase(filePath, CONVERSION_ISSUE_REASON, "https://github.com/yugabyte/yugabyte-db/issues/10866",
+				"Remove it from the exported schema", "CONVERSION", convName, sqlStmtInfo.formattedStmt)
+		} else {
+			//pg_query doesn't seem to have a Node type of AlterConversionStmt so using regex for now
+			if stmt := alterConvRegex.FindStringSubmatch(sqlStmtInfo.stmt); stmt != nil {
+				reportCase(filePath, "ALTER CONVERSION is not supported yet", "https://github.com/YugaByte/yugabyte-db/issues/10866",
+					"Remove it from the exported schema", "CONVERSION", stmt[1], sqlStmtInfo.formattedStmt)
+			}
```

Comment:
just checking for completeness sake: in pg_dump, it's only possible to get a CREATE CONVERSION or a ALTER CONVERSION, right? Nothing else..